.github/workflows/test-package.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     - name: Validate build
       run: |
         python setup.py sdist bdist_wheel
-        pip install dist/rubicon_ml-*.whl
+        pip install dist/rubicon*ml-*.whl
```

Comment:
newer versions of setuptools use the package name with an underscore, where old versions used the library name with a dash. allows for both so we can test build in all available versions